crates/matrix-sdk-ui/tests/integration/room_list_service.rs

```diff
@@ -1476,6 +1476,16 @@ async fn test_dynamic_entries_stream() -> Result<(), Error> {
         push front [ "!r4:bar.org" ];
         end;
     };
+    assert_entries_batch! {
```

Comment:
This commit (824cb0682a072ed73a2c8f1c7d1b92b83ddf34c2) likely tweaks some test expectations after a change in the code, can you fold it into the same commit that introduced the change in test expectations, instead?